<!doctype html><html lang=en><head><meta charset=utf-8><title>Cohen Lab/Journal Club</title><meta content="Papers discussed in the Cohen Lab journal club" name=description><meta content="width=device-width,initial-scale=1" name=viewport><meta content=no-referrer-when-downgrade name=referrer><meta content="Cohen Lab/Journal Club" property=og:title><meta content=website property=og:type><meta content=#f0f0f0 name=theme-color><link href=https://bclab.wustl.edu/main.css rel=stylesheet><link href=https://fonts.googleapis.com rel=preconnect><link crossorigin href=https://fonts.gstatic.com rel=preconnect><link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;800&display=swap" rel=stylesheet><body><div class=page><header><h1><a href=https://bclab.wustl.edu>Cohen Lab</a></h1><nav><ul><li><a href=https://bclab.wustl.edu>Home</a><li><a href=https://bclab.wustl.edu/research>Research</a><li><a href=https://bclab.wustl.edu/publications>Publications</a><li><a href=https://bclab.wustl.edu/posts>Posts</a><li><a href=https://bclab.wustl.edu/people>People</a><li><a class=selected href=https://bclab.wustl.edu/journal-club>Journal Club</a><li><a href=https://bclab.wustl.edu/resources>Resources</a><li><a href=https://bclab.wustl.edu/contact>Contact</a></ul></nav></header><main><h1 id=journal-club>Journal Club</h1><p>The Cohen Lab journal club meets at 12:00 PM on Wednesdays in Couch 4304.<p>Every few weeks, each lab member pitches an article that they have recently read. They have two minutes to sell the lab on the exciting findings and merits of the experiments. At the conclusion, the lab votes on the papers that will populate the coming weeks’ discussions.<h2 id=previously-discussed-papers>Previously-Discussed Papers</h2><div class=section><p><b>6/25/25</b>: Pavan C et al. <a href=https://doi.org/10.1016/j.stem.2025.03.008 rel=noreferrer target=_blank>A cloaked human stem-cell-derived neural graft capable of functional integration and immune evasion in rodent models</a>. <i>Cell Stem Cell</i> (2025) [doi: 10.1016/j.stem.2025.03.008]<p><b>6/18/25</b>: Kong W et al. <a href=https://doi.org/10.1038/s41588-025-02135-z rel=noreferrer target=_blank>Genomic analysis of 1,325 Camellia accessions sheds light on agronomic and metabolic traits for tea plant improvement</a>. <i>Nature Genetics</i> (2025) [doi: 10.1038/s41588-025-02135-z]<p><b>6/4/25</b>: Tong K et al <a href=https://doi.org/10.1038/s41586-025-08689-6 rel=noreferrer target=_blank>Genome duplication in a long-term multicellularity evolution experiment</a>. <i>Nature</i> (2025) [doi: 10.1038/s41586-025-08689-6]<p><b>5/28/25</b>: Michelson DA et al. <a href=https://doi.org/10.1016/j.cell.2022.05.018 rel=noreferrer target=_blank>Thymic epithelial cells co-opt lineage-defining transcription factors to eliminate autoreactive T cells</a>. <i>Cell</i> (2022) [doi: 10.1016/j.cell.2022.05.018]<p><b>5/14/25</b>: Merkenschlager J et al. <a href=https://doi.org/10.1038/s41586-025-08728-2 rel=noreferrer target=_blank>Regulated somatic hypermutation enhances antibody affinity maturation</a>. <i>Nature</i> (2025) [doi: 10.1038/s41586-025-08728-2]<p><b>5/7/25</b>: Zhang C et al. <a href=https://doi.org/10.1038/s41586-024-08040-5 rel=noreferrer target=_blank>Parallel molecular data storage by printing epigenetic bits on DNA</a>. <i>Nature</i> (2024) [doi: 10.1038/s41586-024-08040-5]<p><b>4/23/25</b>: Ji D et al. <a href=https://doi.org/10.1016/j.molcel.2023.11.020 rel=noreferrer target=_blank>FOXA1 forms biomolecular condensates that unpack condensed chromatin to function as a pioneer factor</a>. <i>Molecular Cell</i> (2024) [doi: 10.1016/j.molcel.2023.11.020]<p><b>4/16/25</b>: Vollger MR et al. <a href=https://doi.org/10.1101/2024.06.14.599122 rel=noreferrer target=_blank>A haplotype-resolved view of human gene regulation</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.06.14.599122]<p><b>4/9/25</b>: Zhou BR et al. <a href=https://doi.org/10.1016/j.molcel.2024.07.016 rel=noreferrer target=_blank>Structural insights into the cooperative nucleosome recognition and chromatin opening by FOXA1 and GATA4</a>. <i>Molecular Cell</i> (2024) [doi: 10.1016/j.molcel.2024.07.016]<p><b>4/2/25</b>: Meneu R et al. <a href=https://doi.org/10.1126/science.adm9466 rel=noreferrer target=_blank>Sequence-dependent activity and compartmentalization of foreign DNA in a eukaryotic nucleus</a>. <i>Science</i> (2025) [doi: 10.1126/science.adm9466]<p><b>3/26/25</b>: Johnstone CP et al. <a href=https://doi.org/10.1101/2025.01.19.633652 rel=noreferrer target=_blank>Gene syntax defines supercoiling-mediated transcriptional feedback</a>. <i>bioRxiv</i> (2025) [doi: 10.1101/2025.01.19.633652]<p><b>3/19/25</b>: Gibson TJ et al. <a href=https://doi.org/10.1038/s41594-024-01231-8 rel=noreferrer target=_blank>Protein-intrinsic properties and context-dependent effects regulate pioneer factor binding and function</a>. <i>Nature Structural & Molecular Biology</i> (2024) [doi: 10.1038/s41594-024-01231-8]<p><b>3/12/25</b>: Mahara S et al. <a href=https://doi.org/10.1016/j.molcel.2024.10.005 rel=noreferrer target=_blank>Transient promoter interactions modulate developmental gene activation</a>. <i>Molecular Cell</i> (2024) [doi: 10.1016/j.molcel.2024.10.005]<p><b>2/19/25</b>: Cruz NDL et al. <a href=https://doi.org/10.1016/j.molcel.2024.08.017 rel=noreferrer target=_blank>Disorder-mediated interactions target proteins to specific condensates</a>. <i>Molecular Cell</i> (2024) [doi: 10.1016/j.molcel.2024.08.017]<p><b>2/12/25</b>: Cenik BK et al. <a href=https://doi.org/10.1016/j.molcel.2024.11.007 rel=noreferrer target=_blank>TurboCas: A method for locus-specific labeling of genomic regions and isolating their associated protein interactome</a>. <i>Molecular Cell</i> (2024) [doi: 10.1016/j.molcel.2024.11.007]<p><b>2/5/25</b>: Langstein-Skora I et al. <a href=https://doi.org/10.1101/2022.02.10.480018 rel=noreferrer target=_blank>Sequence- and chemical specificity define the functional landscape of intrinsically disordered regions</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2022.02.10.480018]<p><b>1/22/25</b>: Jensen CL et al. <a href=https://doi.org/10.1016/j.molcel.2024.10.021 rel=noreferrer target=_blank>Long-range regulation of transcription scales with genomic distance in a gene-specific manner</a>. <i>Molecular Cell</i> (2024) [doi: 10.1016/j.molcel.2024.10.021]<p><b>1/15/25</b>: Karpinska MA et al. <a href=https://doi.org/10.1101/2024.11.20.624447 rel=noreferrer target=_blank>CTCF depletion decouples enhancer-mediated gene activation from chromatin hub formation during cellular differentiation</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.11.20.624447]<p><b>12/18/24</b>: Hansen KL et al. <a href=https://doi.org/10.1101/2024.10.04.615095 rel=noreferrer target=_blank>Synergy between cis-regulatory elements can render cohesin dispensable for distal enhancer function</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.10.04.615095]<p><b>12/11/24</b>: Lang TJ et al. <a href=https://doi.org/10.1093/nar/gkae846 rel=noreferrer target=_blank>Massively parallel binding assay (MPBA) reveals limited transcription factor binding cooperativity, challenging models of specificity</a>. <i>Nucleic Acids Research</i> (2024) [doi: 10.1093/nar/gkae846]<p><b>12/4/24</b>: Swanson EG et al. <a href=https://doi.org/10.1101/2024.11.06.622310 rel=noreferrer target=_blank>Deaminase-assisted single-molecule and single-cell chromatin fiber sequencing</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.11.06.622310]<p><b>11/13/24</b>: Chen et al. <a href=https://doi.org/10.7554/eLife.75064 rel=noreferrer target=_blank>Mechanisms governing target search and binding dynamics of hypoxia-inducible factors</a>. <i>eLife</i> (2022) [doi: 10.7554/eLife.75064]<p><b>10/9/24</b>: Durdu et al. <a href=https://doi.org/10.1101/2024.08.05.606702 rel=noreferrer target=_blank>Chromatin-dependent motif syntax defines differentiation trajectories</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.08.05.606702]<p><b>10/2/24</b>: Ma K et al. <a href=https://doi.org/10.1101/2023.07.12.548370 rel=noreferrer target=_blank>Deep Mutational Scanning in Disease-related Genes with Saturation Mutagenesis-Reinforced Functional Assays (SMuRF)</a>. <i>bioRxiv</i> (2023) [doi: 10.1101/2023.07.12.548370]<p><b>9/25/24</b>: Fromel R et al. <a href=https://doi.org/10.1101/2024.08.26.609645 rel=noreferrer target=_blank>Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.08.26.609645]<p><b>9/18/24</b>: Aboreden NG et al. <a href=https://doi.org/10.1101/2024.08.23.609430 rel=noreferrer target=_blank>LDB1 establishes multi-enhancer networks to regulate gene expression</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.08.23.609430]<p><b>9/11/24</b>: Hummel N et al. <a href=https://doi.org/10.1016/j.cels.2024.05.007 rel=noreferrer target=_blank>Systematic identification of transcriptional activation domains from non-transcription factor proteins in plants and yeast</a>. <i>Cell Systems</i> (2024) [doi: 10.1016/j.cels.2024.05.007]<p><b>9/4/24</b>: Tullius TW et al. <a href=https://doi.org/10.1101/2023.12.22.573133 rel=noreferrer target=_blank>RNA polymerases reshape chromatin and coordinate transcription on individual fibers</a>. <i>bioRxiv</i> (2023) [doi: 10.1101/2023.12.22.573133]<p><b>8/21/24</b>: Bower G et al. <a href=https://doi.org/10.1101/2024.05.26.595809 rel=noreferrer target=_blank>Conserved Cis-Acting Range Extender Element Mediates Extreme Long-Range Enhancer Activity in Mammals</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.05.26.595809]<p><b>8/14/24</b>: Perkins ML et al. <a href=https://doi.org/10.1101/2024.06.13.598840 rel=noreferrer target=_blank>Chromatin enables precise and scalable gene regulation with factors of limited specificity</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.06.13.598840]<p><b>8/7/24</b>: Xu C et al. <a href=https://doi.org/10.1016/j.molcel.2024.06.022 rel=noreferrer target=_blank>Systematic dissection of sequence features affecting binding specificity of a pioneer factor reveals binding synergy between FOXA1 and AP-1</a>. <i>Molecular Cell</i> (2024) [doi: 10.1016/j.molcel.2024.06.022]<p><b>7/31/24</b>: Leander M et al. <a href=https://doi.org/10.7554/eLife.79932 rel=noreferrer target=_blank>Deep mutational scanning and machine learning reveal structural and molecular rules governing allosteric hotspots in homologous proteins</a>. <i>eLife</i> (2022) [doi: 10.7554/eLife.79932]<p><b>7/24/24</b>: Le DJ at al. <a href=https://doi.org/10.1101/2024.05.08.593251 rel=noreferrer target=_blank>Super-enhancer interactomes from single cells link clustering and transcription</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.05.08.593251]<p><b>7/10/24</b>: Khetan et al. <a href=https://doi.org/10.1101/2024.03.05.583629 rel=noreferrer target=_blank>Overlapping binding sites underlie TF genomic occupancy</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.03.05.583629]<p><b>6/26/24</b>: Ungerleider et al. <a href=https://doi.org/10.1093/nar/gkae175 rel=noreferrer target=_blank>Viral reprogramming of host transcription initiation</a>. <i>Nucleic Acids Research</i> (2024) [doi: 10.1093/nar/gkae175]<p><b>6/19/24</b>: Grealey et al. <a href=https://doi.org/10.1093/molbev/msac034 rel=noreferrer target=_blank>The Carbon Footprint of Bioinformatics</a>. <i>Molecular Biology and Evolution</i> (2022) [doi: 10.1093/molbev/msac034]<p><b>6/12/24</b>: Bell et al. <a href=https://doi.org/10.1038/s41588-024-01749-z rel=noreferrer target=_blank>Comparative cofactor screens show the influence of transactivation domains and core promoters on the mechanisms of transcription</a>. <i>Nature Genetics</i> (2024) [doi: 10.1038/s41588-024-01749-z]<p><b>6/5/24</b>: Dudnyk et al. <a href=https://doi.org/10.1126/science.adj0116 rel=noreferrer target=_blank>Sequence basis of transcription initiation in the human genome</a>. <i>Science</i> (2024) [doi: 10.1126/science.adj0116]<p><b>5/29/24</b>: Yuan et al. <a href=https://doi.org/10.1038/s41587-024-02182-7 rel=noreferrer target=_blank>Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data</a>. <i>Nature Biotechnology</i> (2024) [doi: 10.1038/s41587-024-02182-7]<p><b>5/15/24</b>: Thomas H et al. <a href=https://doi.org/10.1101/2023.12.06.570399 rel=noreferrer target=_blank>Enhancer cooperativity can compensate for loss of activity over large genomic distances</a>. <i>bioRxiv</i> (2023) [doi: 10.1101/2023.12.06.570399]<p><b>5/1/24</b>: Harvey WT et al. <a href=https://doi.org/10.1101/gr.278070.123 rel=noreferrer target=_blank>Whole-genome long-read sequencing downsampling and its effect on variant-calling precision and recall</a>. <i>Genome Research</i> (2023) [doi: 10.1101/gr.278070.123]<p><b>4/17/24</b>: Camellato BR <a href=https://doi.org/10.1038/s41586-024-07128-2 rel=noreferrer target=_blank>Synthetic reversed sequences reveal default genomic states</a>. <i>Nature</i> (2024) [doi: 10.1038/s41586-024-07128-2]<p><b>4/10/24</b>: Vries TD et al. <a href=https://doi.org/10.1126/sciadv.adm7435 rel=noreferrer target=_blank>Specific protein-RNA interactions are mostly preserved in biomolecular condensates</a>. <i>Science Advances</i> (2024) [doi: 10.1126/sciadv.adm7435]<p><b>4/3/24</b>: Doughty BR et al. <a href=https://doi.org/10.1101/2024.02.02.578660 rel=noreferrer target=_blank>Single-molecule chromatin configurations link transcription factor binding to expression in human cells</a>. <i>bioRxiv</i> (2024) [doi: 10.1101/2024.02.02.578660]<p><b>3/27/24</b>: Xia B et al. <a href=https://doi.org/10.1038/s41586-024-07095-8 rel=noreferrer target=_blank>On the genetic basis of tail-loss evolution in humans and apes</a>. <i>Science</i> (2024) [doi: 10.1038/s41586-024-07095-8]<p><b>2/28/24</b>: Hung T-C et al. <a href=https://doi.org/10.1038/s41588-023-01641-2 rel=noreferrer target=_blank>Boundary stacking interactions enable cross-TAD enhancer–promoter communication during limb development</a>. <i>Nature Genetics</i> (2024) [doi: 10.1038/s41588-023-01641-2]<p><b>2/21/24</b>: Xing Y-H et al. <a href=https://doi.org/10.1038/s41587-023-01737-4 rel=noreferrer target=_blank>DisP-seq reveals the genome-wide functional organization of DNA-associated disordered proteins</a>. <i>Nature Biotechnology</i> (2024) [doi: 10.1038/s41587-023-01737-4]<p><b>2/14/24</b>: Huang K et al. <a href=https://doi.org/10.1101/2023.12.12.571389 rel=noreferrer target=_blank>Sequential Optimal Experimental Design of Perturbation Screens Guided by Multi-modal Priors</a>. <i>bioRxiv</i> (2023) [doi: 10.1101/2023.12.12.571389]<p><b>2/7/24</b>: Kim S et al. <a href=https://doi.org/10.1016/j.cell.2023.12.032 rel=noreferrer target=_blank>DNA-guided transcription factor cooperativity shapes face and limb mesenchyme</a>. <i>Cell</i> (2024) [doi: 10.1016/j.cell.2023.12.032]<p><b>1/3/24</b>: Balasubramanian D et al. <a href=https://doi.org/10.1093/nar/gkad1183 rel=noreferrer target=_blank>Enhancer–promoter interactions can form independently of genomic distance and be functional across TAD boundaries</a>. <i>Nucleic Acid Research</i> (2023) [doi: 10.1093/nar/gkad1183]<p><b>12/20/23</b>: Matthew P.S et al. <a href=https://doi.org/10.1016/j.cell.2023.10.012 rel=noreferrer target=_blank>RNA polymerase II dynamics and mRNA stability feedback scale mRNA amounts with cell size</a>. <i>Cell</i> (2023) [doi: 10.1016/j.cell.2023.10.012]<p><b>12/13/23</b>: Evgenia N et al. <a href=https://doi.org/10.1016/j.cels.2023.09.005 rel=noreferrer target=_blank>Genome-wide measurement of RNA dissociation from chromatin classifies transcripts by their dynamics and reveals rapid dissociation of enhancer lncRNAs</a>. <i>Cell Systems</i> (2023) [doi: 10.1016/j.cels.2023.09.005]<p><b>12/6/23</b>: Ilias G-S et al. <a href=https://doi.org/10.1016/j.xgen.2022.100111 rel=noreferrer target=_blank>High-throughput characterization of the role of non-B DNA motifs on promoter function</a>. <i>Cell Genomics</i> (2022) [doi: 10.1016/j.xgen.2022.100111]<p><b>11/29/23</b>: Andreas R.G et al. <a href=https://doi.org/10.1101/2023.11.09.563812 rel=noreferrer target=_blank>An encyclopedia of enhancer-gene regulatory interactions in the human genome</a>. <i>bioRxiv</i> (2023) [doi: 10.1101/2023.11.09.563812]<p><b>11/8/23</b>: Patil A et al. <a href=https://doi.org/10.1016/j.cell.2023.08.032 rel=noreferrer target=_blank>A disordered region controls cBAF activity via condensation and partner recruitment</a>. <i>Cell</i> (2023) [doi: 10.1016/j.cell.2023.08.032]<p><b>11/1/23</b>: Horton CA et al. <a href=https://doi.org/10.1126/science.add1250 rel=noreferrer target=_blank>Short tandem repeats bind transcription factors to tune eukaryotic gene expression</a>. <i>Science</i> (2023) [doi: 10.1126/science.add1250]<p><b>10/18/23</b>: Pownall ME et al. <a href=https://doi.org/10.1126/science.ade530 rel=noreferrer target=_blank>Chromatin expansion microscopy reveals nanoscale organization of transcription and chromatin</a>. <i>Science</i> (2023) [doi: 10.1126/science.ade530]<p><b>10/11/23</b>: Quililan K et al. <a href=https://doi.org/10.1101/2023.02.27.530214 rel=noreferrer target=_blank>In vitro reconstitution of chromatin domains</a>. <i>bioRxiv</i> (2023) [doi: 10.1101/2023.02.27.530214]<p><b>9/20/23</b>: Hawkins JA et al. <a href=https://doi.org/10.1073/pnas.1802640115 rel=noreferrer target=_blank>Indel-correcting DNA barcodes for high-throughput sequencing</a>. <i>PNAS</i> (2023) [doi: 10.1073/pnas.1802640115]<p><b>9/20/23</b>: Press WH et al. <a href=https://doi.org/10.1093/pnasnexus/pgac252 rel=noreferrer target=_blank>Fast trimer statistics facilitate accurate decoding of large random DNA barcode sets even at large sequencing error rates</a>. <i>PNAS Nexus</i> (2023) [doi: 10.1093/pnasnexus/pgac252]<p><b>9/13/23</b>: Oksuz O et al. <a href=https://doi.org/10.1016/j.molcel.2023.06.012 rel=noreferrer target=_blank>Transcription factors interact with RNA to regulate genes</a>. <i>Molecular Cell</i> (2023) [doi: 10.1016/j.molcel.2023.06.012]<p><b>9/6/23</b>: Jacobs J et al. <a href=https://doi.org/10.1126/science.adf6149 rel=noreferrer target=_blank>Widespread regulatory specificities between transcriptional co-repressors and enhancers in Drosophila</a>. <i>Science</i> (2023) [doi: 10.1126/science.adf6149]<p><b>8/30/23</b>: Goyal Y et al. <a href=https://doi.org/10.1038/s41586-023-06342-8 rel=noreferrer target=_blank>Diverse clonal fates emerge upon drug treatment of homogeneous cancer cells</a>. <i>Nature</i> (2023) [doi: 10.1038/s41586-023-06342-8]<p><b>8/23/23</b>: Jonathan L et al. <a href=https://doi.org/10.1016/j.celrep.2023.112748 rel=noreferrer target=_blank>Different chromatin-scanning modes lead to targeting of compacted chromatin by pioneer factors FOXA1 and SOX2</a>. <i>Cell Reports</i> (2023) [doi: 10.1016/j.celrep.2023.112748]<p><b>8/2/23</b>: Yoon JK et al. <a href=https://doi.org/10.1126/sciadv.adg1123 rel=noreferrer target=_blank>Light-activated macromolecular phase separation modulates transcription by reconfiguring chromatin interactions</a>. <i>Science Advances</i> (2023) [doi: 10.1126/sciadv.adg1123]<p><b>7/26/23</b>: Matthew AB et al. <a href=https://doi.org/10.1016/j.cell.2023.05.004 rel=noreferrer target=_blank>Temperature-dependent RNA editing in octopus extensively recodes the neural proteome</a>. <i>Cell</i> (2023) [doi: 10.1016/j.cell.2023.05.004]<p><b>7/19/23</b>: Nicole D et al. <a href=https://doi.org/10.1038/s41586-023-05906-y rel=noreferrer target=_blank>Large-scale mapping and mutagenesis of human transcriptional effector domains</a>. <i>Nature</i> (2023) [doi: 10.1038/s41586-023-05906-y]<p><b>7/12/23</b>: Koji K et al. <a href=https://doi.org/10.1016/j.molcel.2023.04.018 rel=noreferrer target=_blank>Functional coordination between transcription factor clustering and gene activity</a>. <i>Molecular Cell</i> (2023) [doi: 10.1016/j.molcel.2023.04.018]<p><b>7/12/23</b>: Joseph M et al. <a href=https://doi.org/10.1093/nar/gkad227 rel=noreferrer target=_blank>Transcription factor clusters enable target search but do not contribute to target gene activation</a>. <i>Nucleid Acid Research</i> (2023) [doi: 10.1093/nar/gkad227]<p><b>6/21/23</b>: Song BP et al. <a href=https://doi.org/10.1016/j.celrep.2023.112052 rel=noreferrer target=_blank>Diverse logics and grammar encode notochord enhancers</a>. <i>Cell Reports</i> (2023) [doi: 10.1016/j.celrep.2023.112052]<p><b>6/14/23</b>: Zhou Z et al. <a href=https://doi.org/10.1126/science.add7631 rel=noreferrer target=_blank>Engineering longevity—design of a synthetic gene oscillator to slow cellular aging</a>. <i>Science</i> (2023) [doi: 10.1126/science.add7631]<p><b>6/7/23</b>: Kumar DBU et al. <a href=https://doi.org/10.3389/fmolb.2023.1110445 rel=noreferrer target=_blank>A genome–wide CRISPR activation screen identifies SCREEM a novel SNAI1 super-enhancer demarcated by eRNAs</a>. <i>Frontiers in Molecular Biosicences</i> (2023) [doi: 10.3389/fmolb.2023.1110445]<p><b>5/31/23</b>: Naqvi S et al. <a href=https://doi.org/10.1038/s41588-023-01366-2 rel=noreferrer target=_blank>Precise modulation of transcription factor level/s identifies features underlying dosage sensitivity</a>. <i>Nature Genetics</i> (2023) [doi: 10.1038/s41588-023-01366-2]<p><b>3/15/23</b>: Kim YJ et al. <a href=https://doi.org/10.7554/eLife.73395 rel=noreferrer target=_blank>Predictive modeling reveals that higher-order cooperativity drives transcriptional repression in a synthetic developmental enhancer</a>. <i>eLife</i> (2022) [doi: 10.7554/eLife.73395]<p><b>3/8/23</b>: Chakraborty S et al. <a href=https://doi.org/10.1038/s41588-022-01295-6 rel=noreferrer target=_blank>Enhancer–promoter interactions can bypass CTCF-mediated boundaries and contribute to phenotypic robustness</a>. <i>Nature Genetics</i> (2023) [doi: 10.1038/s41588-022-01295-6]<p><b>2/22/23</b>: Anderson AG et al. <a href=https://doi.org/10.1016/j.xgen.2023.100263 rel=noreferrer target=_blank>Single nucleus multiomics identifies ZEB1 and MAFB as candidate regulators of Alzheimer’s disease-specific cis-regulatory elements</a>. <i>Cell Genomics</i> (2023) [doi: 10.1016/j.xgen.2023.100263]<p><b>2/15/23</b>: Yanchus C et al. <a href=https://doi.org/10.1126/science.abj2890 rel=noreferrer target=_blank>A noncoding single-nucleotide polymorphism at 8q24 drives IDH1-mutant glioma formation</a>. <i>Science</i> (2022) [doi: 10.1126/science.abj2890]<p><b>1/25/23</b>: Economou A et al. <a href=https://doi.org/10.1016/j.devcel.2022.11.008 rel=noreferrer target=_blank>Nodal signaling establishes a competency window for stochastic cell fate switching</a>. <i>Developmental Cell</i> (2022) [doi: 10.1016/j.devcel.2022.11.008]<p><b>1/11/23</b>: Hsieh TS et al. <a href=https://doi.org/10.1038/s41588-022-01223-8 rel=noreferrer target=_blank>Enhancer–promoter interactions and transcription are largely maintained upon acute loss of CTCF, cohesin, WAPL or YY1</a>. <i>Nature Genetics</i> (2022) [doi: 10.1038/s41588-022-01223-8]<p><b>1/4/23</b>: Daniels KG et al. <a href=https://doi.org/10.1126/science.abq0225 rel=noreferrer target=_blank>Decoding CAR T cell phenotype using combinatorial signaling motif libraries and machine learning</a>. <i>Science</i> (2022) [doi: 10.1126/science.abq0225]<p><b>12/22/22</b>: Lalanne JB et al. <a href=https://doi.org/10.1101/2022.12.10.519236 rel=noreferrer target=_blank>Multiplex profiling of developmental enhancers with quantitative, single-cell expression reporters</a>. <i>bioRxiv</i> (2022) [doi: 10.1101/2022.12.10.519236]<p><b>11/9/22</b>: Ray et al. <a href=https://doi.org/10.1038/s41586-022-05217-8 rel=noreferrer target=_blank>A mechanism for oxidative damage repair at gene regulatory elements</a>. <i>Nature</i> (2022) [doi: 10.1038/s41586-022-05217-8]<p><b>11/2/22</b>: Morin et al. <a href=https://doi.org/10.1038/s41567-021-01462-2 rel=noreferrer target=_blank>Sequence-dependent surface condensation of a pioneer transcription factor on DNA</a>. <i>Nature Physics</i> (2022) [doi: 10.1038/s41567-021-01462-2]<p><b>10/5/22</b>: Asimi V et al. <a href=https://doi.org/10.1038/s41588-022-01132-w rel=noreferrer target=_blank>Hijacking of transcriptional condensates by endogenous retroviruses</a>. <i>Nature Genetics</i> (2022) [doi: 10.1038/s41588-022-01132-w]<p><b>9/14/22</b>: Lin X et al. <a href=https://doi.org/10.1126/science.abk3512 rel=noreferrer target=_blank>Nested epistasis enhancer networks for robust genome regulation</a>. <i>Science</i> (2022) [doi: 10.1126/science.abk3512]<p><b>9/7/22</b>: Pachano T et al. <a href=https://doi.org/10.1038/s41588-021-00888-x rel=noreferrer target=_blank>Orphan CpG islands amplify poised enhancer regulatory activity and determine target gene responsiveness</a>. <i>Nature Genetics</i> (2021) [doi: 10.1038/s41588-021-00888-x]<p><b>8/31/22</b>: Neumayr C et al. <a href=https://doi.org/10.1038/s41586-022-04779-x rel=noreferrer target=_blank>Differential cofactor dependencies define distinct types of human enhancers</a>. <i>Nature</i> (2022) [doi: 10.1038/s41586-022-04779-x]<p><b>8/17/22</b>: Bergman DT et al. <a href=https://doi.org/10.1038/s41586-022-04877-w rel=noreferrer target=_blank>Compatibility rules of human enhancer and promoter sequences</a>. <i>Nature</i> (2022) [doi: 10.1038/s41586-022-04877-w]<p><b>8/10/22</b>: Bal E et al. <a href=https://doi.org/10.1038/s41586-022-04906-8 rel=noreferrer target=_blank>Super-enhancer hypermutation alters oncogene expression in B cell lymphoma</a>. <i>Nature</i> (2022) [doi: 10.1038/s41586-022-04906-8]<p><b>8/3/22</b>: Choi J et al. <a href=https://doi.org/10.1038/s41586-022-04922-8 rel=noreferrer target=_blank>A time-resolved, multi-symbol molecular recorder via sequential genome editing</a>. <i>Nature</i> (2022) [doi: 10.1038/s41586-022-04922-8]<p><b>7/13/22</b>: Konstantinides N et al. <a href=https://doi.org/10.1038/s41586-022-04564-w rel=noreferrer target=_blank>A complete temporal transcription factor series in the fly visual system</a>. <i>Nature</i> (2022) [doi: 10.1038/s41586-022-04564-w]<p><b>7/6/22</b>: Snetkova V et al. <a href=https://doi.org/10.1038/s41588-021-00812-3 rel=noreferrer target=_blank>Ultraconserved enhancer function does not require perfect sequence conservation</a>. <i>Nature Genetics</i> (2021) [doi: 10.1038/s41588-021-00812-3]<p><b>6/8/22</b>: Ma Y et al. <a href=https://doi.org/10.1016/j.cell.2022.01.026 rel=noreferrer target=_blank>Synthetic mammalian signaling circuits for robust cell population control</a>. <i>Cell</i> (2022) [doi: 10.1016/j.cell.2022.01.026]<p><b>5/18/22</b>: Topolewski P et al. <a href=https://doi.org/10.1126/scisignal.abd9303 rel=noreferrer target=_blank>Phenotypic variability, not noise, accounts for most of the cell-to-cell heterogeneity in IFN-gamma and oncostatin M signaling responses</a>. <i>Science Signaling</i> (2022) [doi: 10.1126/scisignal.abd9303]<p><b>4/19/22</b>: Marklund E et al. <a href=https://doi.org/10.1126/science.abg7427 rel=noreferrer target=_blank>Sequence specificity in DNA binding is mainly governed by association</a>. <i>Science</i> (2022) [doi: 10.1126/science.abg7427]<p><b>4/13/22</b>: Rao S et al. <a href=https://doi.org/10.1016/j.molcel.2021.02.014 rel=noreferrer target=_blank>Cooperative binding between distant transcription factors is a hallmark of active enhancers</a>. <i>Molecular Cell</i> (2021) [doi: 10.1016/j.molcel.2021.02.014]<p><b>4/6/22</b>: Traets JJH et al. <a href=https://doi.org/10.7554/eLife.66955 rel=noreferrer target=_blank>Mechanism of life-long maintenance of neuron identity despite molecular fluctuations</a>. <i>eLife</i> (2021) [doi: 10.7554/eLife.66955]<p><b>3/23/22</b>: Ursu O et al. <a href=https://doi.org/10.1038/s41587-021-01160-7 rel=noreferrer target=_blank>Massively parallel phenotyping of coding variants in cancer with Perturb-seq</a>. <i>Nature Biotechnology</i> (2022) [doi: 10.1038/s41587-021-01160-7]<p><b>2/23/22</b>: Seczynska M et al. <a href=https://doi.org/10.1038/s41586-021-04228-1 rel=noreferrer target=_blank>Genome surveillance by HUSH-mediated silencing of intronless mobile elements</a>. <i>Nature</i> (2022) [doi: 10.1038/s41586-021-04228-1]<p><b>2/2/22</b>: Duveau F et al. <a href=https://doi.org/10.7554/eLife.67806 rel=noreferrer target=_blank>Mutational sources of trans-regulatory variation affecting gene expression in Saccharomyces cerevisiae</a>. <i>eLife</i> (2021) [doi: 10.7554/eLife.67806]<p><b>1/26/22</b>: Bintu L et al. <a href=https://doi.org/10.1126/science.aab2956 rel=noreferrer target=_blank>Dynamics of epigenetic regulation at the single-cell level</a>. <i>Science</i> (2016) [doi: 10.1126/science.aab2956]<p><b>12/8/21</b>: Hocker JD et al. <a href=https://doi.org/10.1126/sciadv.abf1444 rel=noreferrer target=_blank>Cardiac cell type-specific gene regulatory programs and disease risk association</a>. <i>Science Advances</i> (2021) [doi: 10.1126/sciadv.abf1444]<p><b>11/10/21</b>: Rinzema NJ et al. <a href=https://doi.org/10.1101/2021.10.05.463209 rel=noreferrer target=_blank>Building regulatory landscapes: enhancer recruits cohesin to create contact domains, engage CTCF sites and activate distant genes</a>. <i>bioRχiv Preprint</i> (2021) [doi: 10.1101/2021.10.05.463209]<p><b>11/3/21</b>: de Almeida BP et al. <a href=https://doi.org/10.1101/2021.10.05.463203 rel=noreferrer target=_blank>DeepSTARR predicts enhancer activity from DNA sequence and enables the de novo design of enhancers</a>. <i>bioRχiv Preprint</i> (2021) [doi: 10.1101/2021.10.05.463203]<p><b>10/27/21</b>: Li YE et al. <a href=https://doi.org/10.1038/s41586-021-03604-1 rel=noreferrer target=_blank>An atlas of gene regulatory elements in adult mouse cerebrum</a>. <i>Nature</i> (2021) [doi: 10.1038/s41586-021-03604-1]<p><b>10/20/21</b>: Garcia DA et al. <a href=https://doi.org/10.1016/j.molcel.2021.01.013 rel=noreferrer target=_blank>An intrinsically disordered region-mediated confinement state contributes to the dynamics and function of transcription factors</a>. <i>Molecular Cell</i> (2021) [doi: 10.1016/j.molcel.2021.01.013]<p><b>10/6/21</b>: Child MB et al. <a href=https://doi.org/10.7554/eLife.64412 rel=noreferrer target=_blank>Live imaging and biophysical modeling support a button-based mechanism of somatic homolog pairing in Drosophila</a>. <i>eLife</i> (2021) [doi: 10.7554/eLife.64412]<p><b>9/29/21</b>: Wan YH et al. <a href=https://doi.org/10.1016/j.cell.2021.04.012 rel=noreferrer target=_blank>Dynamic imaging of nascent RNA reveals general principles of transcription dynamics and stochastic splice site selection</a>. <i>Cell</i> (2021) [doi: 10.1016/j.cell.2021.04.012]<p><b>9/8/21</b>: Zhang YN et al. <a href=https://doi.org/10.1101/gr.275145.120 rel=noreferrer target=_blank>Competition for DNA binding between paralogous transcription factors determines their genomic occupancy and regulatory functions</a>. <i>Genome Research</i> (2021) [doi: 10.1101/gr.275145.120]<p><b>9/1/21</b>: Desai RV et al. <a href=https://doi.org/10.1126/science.abc6506 rel=noreferrer target=_blank>A DNA repair pathway can regulate transcriptional noise to promote cell fate transitions</a>. <i>Science</i> (2021) [doi: 10.1126/science.abc6506]<p><b>8/25/21</b>: Soochit W et al. <a href=https://doi.org/10.1038/s41556-021-00722-w rel=noreferrer target=_blank>CTCF chromatin residence time controls three-dimensional genome organization, gene expression and DNA methylation in pluripotent cells</a>. <i>Nature Cell Biology</i> (2021) [doi: 10.1038/s41556-021-00722-w]<p><b>8/18/21</b>: Tak YE et al. <a href=https://doi.org/10.1038/s41592-021-01224-1 rel=noreferrer target=_blank>Augmenting and directing long-range CRISPR-mediated activation in human cells</a>. <i>Nature Methods</i> (2021) [doi: 10.1038/s41592-021-01224-1]<p><b>7/21/21</b>: Sanford EM et al. <a href=https://doi.org/10.7554/eLife.59388 rel=noreferrer target=_blank>Gene regulation gravitates toward either addition or multiplication when combining the effects of two signals</a>. <i>eLife</i> (2020) [doi: 10.7554/eLife.59388]<p><b>7/14/21</b>: Sonmezer C et al. <a href=https://doi.org/10.1016/j.molcel.2020.11.015 rel=noreferrer target=_blank>Molecular Co-occupancy Identifies Transcription Factor Binding Cooperativity In Vivo</a>. <i>Molecular Cell</i> (2021) [doi: 10.1016/j.molcel.2020.11.015]<p><b>7/7/21</b>: Tycko J et al. <a href=https://doi.org/10.1016/j.cell.2020.11.024 rel=noreferrer target=_blank>High-Throughput Discovery and Characterization of Human Transcriptional Effectors</a>. <i>Cell</i> (2020) [doi: 10.1016/j.cell.2020.11.024]<p><b>6/30/21</b>: Ng AHM et al. <a href=https://doi.org/10.1038/s41587-020-0742-6 rel=noreferrer target=_blank>A comprehensive library of human transcription factors for cell fate engineering</a>. <i>Nature Biotechnology</i> (2021) [doi: 10.1038/s41587-020-0742-6]<p><b>6/23/21</b>: Li JR et al. <a href=https://doi.org/10.1038/s41594-020-0493-6 rel=noreferrer target=_blank>Single-gene imaging links genome topology, promoter-enhancer communication and transcription control</a>. <i>Nature Structural & Molecular Biology</i> (2020) [doi: 10.1038/s41594-020-0493-6]<p><b>5/12/21</b>: Hanna RE et al. <a href=https://doi.org/10.1016/j.cell.2021.01.012 rel=noreferrer target=_blank>Massively parallel assessment of human variants with base editor screens</a>. <i>Cell</i> (2021) [doi: 10.1016/j.cell.2021.01.012]<p><b>5/5/21</b>: Kharerin H, Bai L <a href=https://doi.org/10.1371/journal.pcbi.1008560 rel=noreferrer target=_blank>Thermodynamic modeling of genome-wide nucleosome depleted regions in yeast</a>. <i>PLOS Computational Biology</i> (2021) [doi: 10.1371/journal.pcbi.1008560]<p><b>4/28/21</b>: Takei Y et al. <a href=https://doi.org/10.1038/s41586-020-03126-2 rel=noreferrer target=_blank>Integrated spatial genomics reveals global architecture of single nuclei</a>. <i>Nature</i> (2021) [doi: 10.1038/s41586-020-03126-2]<p><b>4/21/21</b>: Emert BL et al. <a href=https://doi.org/10.1038/s41587-021-00837-3 rel=noreferrer target=_blank>Variability within rare cell states enables multiple paths toward drug resistance</a>. <i>Nature Biotechnology</i> (2021) [doi: 10.1038/s41587-021-00837-3]<p><b>4/14/21</b>: Choi J et al. <a href=https://doi.org/10.7554/eLife.65381 rel=noreferrer target=_blank>Evidence for additive and synergistic action of mammalian enhancers during cell fate determination</a>. <i>eLife</i> (2021) [doi: 10.7554/eLife.65381]<p><b>3/3/21</b>: Henninger JE et al. <a href=https://doi.org/10.1016/j.cell.2020.11.030 rel=noreferrer target=_blank>RNA-Mediated Feedback Control of Transcriptional Condensates</a>. <i>Cell</i> (2021) [doi: 10.1016/j.cell.2020.11.030]<p><b>2/24/21</b>: Thomas HF et al. <a href=https://doi.org/10.1016/j.molcel.2020.12.047 rel=noreferrer target=_blank>Temporal dissection of an enhancer cluster reveals distinct temporal and functional contributions of individual elements</a>. <i>Molecular Cell</i> (2021) [doi: 10.1016/j.molcel.2020.12.047]<p><b>2/17/21</b>: Tanaka H et al. <a href=https://doi.org/10.1038/s41467-020-17959-y rel=noreferrer target=_blank>Interaction of the pioneer transcription factor GATA3 with nucleosomes</a>. <i>Nature Communications</i> (2020) [doi: 10.1038/s41467-020-17959-y]<p><b>2/10/21</b>: Akdemir KC et al. <a href=https://doi.org/10.1038/s41588-020-0708-0 rel=noreferrer target=_blank>Somatic mutation distributions in cancer genomes vary with three-dimensional chromatin structure</a>. <i>Nature Genetics</i> (2020) [doi: 10.1038/s41588-020-0708-0]<p><b>2/3/21</b>: Klosin A et al. <a href=https://doi.org/10.1126/science.aav6691 rel=noreferrer target=_blank>Phase separation provides a mechanism to reduce noise in cells</a>. <i>Science</i> (2020) [doi: 10.1126/science.aav6691]<p><b>1/20/21</b>: Bergen V et al. <a href=https://doi.org/10.1038/s41587-020-0591-3 rel=noreferrer target=_blank>Generalizing RNA velocity to transient cell states through dynamical modeling</a>. <i>Nature Biotechnology</i> (2020) [doi: 10.1038/s41587-020-0591-3]<p><b>1/6/21</b>: Domcke S et al. <a href=https://doi.org/10.1126/science.aba7612 rel=noreferrer target=_blank>A human cell atlas of fetal chromatin accessibility</a>. <i>Science</i> (2020) [doi: 10.1126/science.aba7612]<p><b>12/16/20</b>: Wong ES et al. <a href=https://doi.org/10.1126/science.aax8137 rel=noreferrer target=_blank>Deep conservation of the enhancer regulatory code in animals</a>. <i>Science</i> (2020) [doi: 10.1126/science.aax8137]<p><b>12/9/20</b>: Kent S et al. <a href=https://doi.org/10.1016/j.celrep.2020.108248 rel=noreferrer target=_blank>Phase-Separated Transcriptional Condensates Accelerate Target-Search Process Revealed by Live-Cell Single-Molecule Imaging</a>. <i>Cell Reports</i> (2020) [doi: 10.1016/j.celrep.2020.108248]<p><b>11/11/20</b>: Ballare C et al. <a href=https://doi.org/10.1016/j.molcel.2012.10.019 rel=noreferrer target=_blank>Nucleosome-Driven Transcription Factor Binding and Gene Regulation</a>. <i>Molecular Cell</i> (2013) [doi: 10.1016/j.molcel.2012.10.019]<p><b>11/4/20</b>: Ireland WT et al. <a href=https://doi.org/10.7554/eLife.55308 rel=noreferrer target=_blank>Deciphering the regulatory genome of Escherichia coli, one hundred promoters at a time</a>. <i>eLife</i> (2020) [doi: 10.7554/eLife.55308]<p><b>10/21/20</b>: Chiasson MA et al. <a href=https://doi.org/10.7554/eLife.58026 rel=noreferrer target=_blank>Multiplexed measurement of variant abundance and activity reveals VKOR topology, active site and human variant impact</a>. <i>eLife</i> (2020) [doi: 10.7554/eLife.58026]<p><b>10/14/20</b>: Houri-Zeevi L et al. <a href=https://doi.org/10.1016/j.cell.2020.07.022 rel=noreferrer target=_blank>Three Rules Explain Transgenerational Small RNA Inheritance in C. elegans</a>. <i>Cell</i> (2020) [doi: 10.1016/j.cell.2020.07.022]<p><b>10/7/20</b>: Chen SY et al. <a href=https://doi.org/10.1016/j.cels.2020.08.009 rel=noreferrer target=_blank>Optogenetic Control Reveals Differential Promoter Interpretation of Transcription Factor Nuclear Translocation Dynamics</a>. <i>Cell Systems</i> (2020) [doi: 10.1016/j.cels.2020.08.009]<p><b>9/23/20</b>: Jaeger MG et al. <a href=https://doi.org/10.1038/s41588-020-0635-0 rel=noreferrer target=_blank>Selective Mediator dependence of cell-type-specifying transcription</a>. <i>Nature Genetics</i> (2020) [doi: 10.1038/s41588-020-0635-0]<p><b>9/16/20</b>: Sigalova OM et al. <a href=https://doi.org/10.15252/msb.20209539 rel=noreferrer target=_blank>Predictive features of gene expression variation reveal mechanistic link with differential expression</a>. <i>Molecular Systems Biology</i> (2020) [doi: 10.15252/msb.20209539]<p><b>9/9/20</b>: Hoppe C et al. <a href=https://doi.org/10.1016/j.devcel.2020.07.007 rel=noreferrer target=_blank>Modulation of the Promoter Activation Rate Dictates the Transcriptional Response to Graded BMP Signaling Levels in the Drosophila Embryo</a>. <i>Developmental Cell</i> (2020) [doi: 10.1016/j.devcel.2020.07.007]<p><b>9/2/20</b>: Hafner A et al. <a href=https://doi.org/10.1016/j.cels.2020.05.001 rel=noreferrer target=_blank>Quantifying the Central Dogma in the p53 Pathway in Live Single Cells</a>. <i>Cell Systems</i> (2020) [doi: 10.1016/j.cels.2020.05.001]<p><b>8/26/20</b>: Partridge EC et al. <a href=https://doi.org/10.1038/s41586-020-2023-4 rel=noreferrer target=_blank>Occupancy maps of 208 chromatin-associated proteins in one human cell type</a>. <i>Nature</i> (2020) [doi: 10.1038/s41586-020-2023-4]<p><b>8/12/20</b>: Quintero-Cadena P et al. <a href=https://doi.org/10.1016/j.molcel.2020.05.030 rel=noreferrer target=_blank>RNA Pol II Length and Disorder Enable Cooperative Scaling of Transcriptional Bursting</a>. <i>Molecular Cell</i> (2020) [doi: 10.1016/j.molcel.2020.05.030]<p><b>8/5/20</b>: Fitz J et al. <a href=https://doi.org/10.1038/s41588-020-0605-6 rel=noreferrer target=_blank>Spt5-mediated enhancer transcription directly couples enhancer activation with physical promoter interaction</a>. <i>Nature Genetics</i> (2020) [doi: 10.1038/s41588-020-0605-6]<p><b>7/29/20</b>: Long YC et al. <a href=https://doi.org/10.1038/s41588-020-0662-x rel=noreferrer target=_blank>RNA is essential for PRC2 chromatin occupancy and function in human pluripotent stem cells</a>. <i>Nature Genetics</i> (2020) [doi: 10.1038/s41588-020-0662-x]<p><b>7/22/20</b>: Ramaker RC et al. <a href=https://doi.org/10.1101/gr.260463.119 rel=noreferrer target=_blank>Dissecting the regulatory activity and sequence content of loci with exceptional numbers of transcription factor associations</a>. <i>Genome Research</i> (2020) [doi: 10.1101/gr.260463.119]<p><b>7/15/20</b>: Wheat JC et al. <a href=https://doi.org/10.1038/s41586-020-2432-4 rel=noreferrer target=_blank>Single-molecule imaging of transcription dynamics in somatic stem cells</a>. <i>Nature</i> (2020) [doi: 10.1038/s41586-020-2432-4]<p><b>7/1/20</b>: Jost M et al. <a href=https://doi.org/10.1038/s41587-019-0387-5 rel=noreferrer target=_blank>Titrating gene expression using libraries of systematically attenuated CRISPR guide RNAs</a>. <i>Nature Biotechnology</i> (2020) [doi: 10.1038/s41587-019-0387-5]<p><b>6/24/20</b>: Fulco CP et al. <a href=https://doi.org/10.1038/s41588-019-0538-0 rel=noreferrer target=_blank>Activity-by-contact model of enhancer-promoter regulation from thousands of CRISPR perturbations</a>. <i>Nature Genetics</i> (2019) [doi: 10.1038/s41588-019-0538-0]<p><b>6/17/20</b>: Tsujimura T et al. <a href=https://doi.org/10.7554/eLife.47980 rel=noreferrer target=_blank>Controlling gene activation by enhancers through a drug-inducible topological insulator</a>. <i>eLife</i> (2020) [doi: 10.7554/eLife.47980]<p><b>5/27/20</b>: Anderson C et al. <a href=https://doi.org/10.7554/eLife.29593 rel=noreferrer target=_blank>Natural variation in stochastic photoreceptor specification and color preference in Drosophila</a>. <i>eLife</i> (2017) [doi: 10.7554/eLife.29593]<p><b>5/20/20</b>: Bialek W et al. <a href=https://doi.org/10.48550/arXiv.1912.08579 rel=noreferrer target=_blank>Action at a distance in transcriptional regulation</a>. <i>arΧiv Preprint</i> (2019) [doi: 10.48550/arXiv.1912.08579]<p><b>5/13/20</b>: Slobodin B et al. <a href=https://doi.org/10.1016/j.molcel.2020.03.022 rel=noreferrer target=_blank>Transcription Dynamics Regulate Poly(A) Tails and Expression of the RNA Degradation Machinery to Balance mRNA Levels</a>. <i>Molecular Cell</i> (2020) [doi: 10.1016/j.molcel.2020.03.022]<p><b>5/6/20</b>: Poramba-Liyanage DW et al. <a href=https://doi.org/10.1101/gr.256255.119 rel=noreferrer target=_blank>Inhibition of transcription leads to rewiring of locus-specific chromatin proteomes</a>. <i>Genome Research</i> (2020) [doi: 10.1101/gr.256255.119]<p><b>4/15/20</b>: Hackett SR et al. <a href=https://doi.org/10.15252/msb.20199174 rel=noreferrer target=_blank>Learning causal networks using inducible transcription factors and transcriptome-wide time series</a>. <i>Molecular Systems Biology</i> (2020) [doi: 10.15252/msb.20199174]<p><b>4/8/20</b>: Chen J et al. <a href=https://doi.org/10.1126/science.aay0262 rel=noreferrer target=_blank>Pervasive functional translation of noncanonical human open reading frames</a>. <i>Science</i> (2020) [doi: 10.1126/science.aay0262]<p><b>4/1/20</b>: Yokoshi M et al. <a href=https://doi.org/10.1016/j.molcel.2020.02.007 rel=noreferrer target=_blank>Visualizing the Role of Boundary Elements in Enhancer-Promoter Communication</a>. <i>Molecular Cell</i> (2020) [doi: 10.1016/j.molcel.2020.02.007]<p><b>3/4/20</b>: Palii CG et al. <a href=https://doi.org/10.1016/j.stem.2019.02.006 rel=noreferrer target=_blank>Single-Cell Proteomics Reveal that Quantitative Changes in Co-expressed Lineage-Specific Transcription Factors Determine Cell Fate</a>. <i>Cell Stem Cell</i> (2019) [doi: 10.1016/j.stem.2019.02.006]<p><b>2/19/20</b>: Hendy O et al. <a href=https://doi.org/10.1091/mbc.E17-06-0408 rel=noreferrer target=_blank>Differential context-specific impact of individual core promoter elements on transcriptional dynamics</a>. <i>Molecular Biology Of The Cell</i> (2017) [doi: 10.1091/mbc.E17-06-0408]<p><b>2/5/20</b>: Shi Z et al. <a href=https://doi.org/10.1016/j.molcel.2017.05.021 rel=noreferrer target=_blank>Heterogeneous Ribosomes Preferentially Translate Distinct Subpools of mRNAs Genome-wide</a>. <i>Molecular Cell</i> (2017) [doi: 10.1016/j.molcel.2017.05.021]<p><b>1/29/20</b>: Davidson IF et al. <a href=https://doi.org/10.1126/science.aaz3418 rel=noreferrer target=_blank>DNA loop extrusion by human cohesin</a>. <i>Science</i> (2019) [doi: 10.1126/science.aaz3418]<p><b>1/22/20</b>: McFaline-Figueroa JL et al. <a href=https://doi.org/10.1038/s41588-019-0489-5 rel=noreferrer target=_blank>A pooled single-cell genetic screen identifies regulatory checkpoints in the continuum of the epithelial-to-mesenchymal transition</a>. <i>Nature Genetics</i> (2019) [doi: 10.1038/s41588-019-0489-5]<p><b>1/15/20</b>: Feldman D et al. <a href=https://doi.org/10.1016/j.cell.2019.09.016 rel=noreferrer target=_blank>Optical Pooled Screens in Human Cells</a>. <i>Cell</i> (2019) [doi: 10.1016/j.cell.2019.09.016]<p><b>12/11/19</b>: Gibson BA et al. <a href=https://doi.org/10.1016/j.cell.2019.08.037 rel=noreferrer target=_blank>Organization of Chromatin by Intrinsic and Regulated Phase Separation</a>. <i>Cell</i> (2019) [doi: 10.1016/j.cell.2019.08.037]<p><b>12/4/19</b>: Viets K et al. <a href=https://doi.org/10.1016/j.devcel.2019.09.007 rel=noreferrer target=_blank>Characterization of Button Loci that Promote Homologous Chromosome Pairing and Cell-Type-Specific Interchromosomal Gene Regulation</a>. <i>Developmental Cell</i> (2019) [doi: 10.1016/j.devcel.2019.09.007]<p><b>11/20/19</b>: Bartman CR et al. <a href=https://doi.org/10.1016/j.molcel.2018.11.004 rel=noreferrer target=_blank>Transcriptional Burst Initiation and Polymerase Pause Release Are Key Control Points of Transcriptional Regulation</a>. <i>Molecular Cell</i> (2019) [doi: 10.1016/j.molcel.2018.11.004]<p><b>11/13/19</b>: Lord ND et al. <a href=https://doi.org/10.1126/science.aaw4506 rel=noreferrer target=_blank>Stochastic antagonism between two proteins governs a bacterial cell fate switch</a>. <i>Science</i> (2019) [doi: 10.1126/science.aaw4506]<p><b>11/6/19</b>: Huertas J et al. <a href=https://doi.org/10.1016/j.bpj.2019.12.038 rel=noreferrer target=_blank>Nucleosomal DNA Dynamics Mediate Oct4 Pioneer Factor Binding</a>. <i>Biophysical Journal</i> (2020) [doi: 10.1016/j.bpj.2019.12.038]<p><b>10/2/19</b>: Lim B et al. <a href=https://doi.org/10.1016/j.molcel.2018.02.029 rel=noreferrer target=_blank>Visualization of Transvection in Living Drosophila Embryos</a>. <i>Molecular Cell</i> (2018) [doi: 10.1016/j.molcel.2018.02.029]<p><b>9/25/19</b>: Benabdallah NS et al. <a href=https://doi.org/10.1016/j.molcel.2019.07.038 rel=noreferrer target=_blank>Decreased Enhancer-Promoter Proximity Accompanying Enhancer Activation</a>. <i>Molecular Cell</i> (2019) [doi: 10.1016/j.molcel.2019.07.038]<p><b>8/28/19</b>: Tsai A et al. <a href=https://doi.org/10.7554/eLife.45325 rel=noreferrer target=_blank>Multi-enhancer transcriptional hubs confer phenotypic robustness</a>. <i>eLife</i> (2019) [doi: 10.7554/eLife.45325]<p><b>8/21/19</b>: Jin Y et al. <a href=https://doi.org/10.1016/j.cell.2017.07.006 rel=noreferrer target=_blank>The Ground State and Evolution of Promoter Region Directionality</a>. <i>Cell</i> (2017) [doi: 10.1016/j.cell.2017.07.006]<p><b>8/7/19</b>: Wilson MD et al. <a href=https://doi.org/10.1126/science.1160930 rel=noreferrer target=_blank>Species-specific transcription in mice carrying human chromosome 21</a>. <i>Science</i> (2008) [doi: 10.1126/science.1160930]<p><b>7/31/19</b>: Ghavi-Helm Y et al. <a href=https://doi.org/10.1038/s41588-019-0462-3 rel=noreferrer target=_blank>Highly rearranged chromosomes reveal uncoupling between genome topology and gene expression</a>. <i>Nature Genetics</i> (2019) [doi: 10.1038/s41588-019-0462-3]<p><b>7/17/19</b>: Schmiedel JM, Lehner B <a href=https://doi.org/10.1038/s41588-019-0431-x rel=noreferrer target=_blank>Determining protein structures using deep mutagenesis</a>. <i>Nature Genetics</i> (2019) [doi: 10.1038/s41588-019-0431-x]<p><b>7/10/19</b>: Falk M et al. <a href=https://doi.org/10.1038/s41586-019-1275-3 rel=noreferrer target=_blank>Heterochromatin drives compartmentalization of inverted and conventional nuclei</a>. <i>Nature</i> (2019) [doi: 10.1038/s41586-019-1275-3]<p><b>7/3/19</b>: Paliou C et al. <a href=https://doi.org/10.1073/pnas.1900672116 rel=noreferrer target=_blank>Preformed chromatin topology assists transcriptional robustness of Shh during limb development</a>. <i>PNAS</i> (2019) [doi: 10.1073/pnas.1900672116]<p><b>6/19/19</b>: Haberle V et al. <a href=https://doi.org/10.1038/s41586-019-1210-7 rel=noreferrer target=_blank>Transcriptional cofactors display specificity for distinct types of core promoters</a>. <i>Nature</i> (2019) [doi: 10.1038/s41586-019-1210-7]<p><b>6/12/19</b>: McSwiggen DT et al. <a href=https://doi.org/10.7554/eLife.47098 rel=noreferrer target=_blank>Evidence for DNA-mediated nuclear compartmentalization distinct from phase separation</a>. <i>eLife</i> (2019) [doi: 10.7554/eLife.47098]<p><b>5/29/19</b>: Yamada S et al. <a href=https://doi.org/10.1016/j.cub.2019.03.019 rel=noreferrer target=_blank>The Drosophila Pioneer Factor Zelda Modulates the Nuclear Microenvironment of a Dorsal Target Enhancer to Potentiate Transcriptional Output</a>. <i>Current Biology</i> (2019) [doi: 10.1016/j.cub.2019.03.019]<p><b>5/22/19</b>: Bashor CJ et al. <a href=https://doi.org/10.1126/science.aau8287 rel=noreferrer target=_blank>Complex signal processing in synthetic gene circuits using cooperative regulatory assemblies</a>. <i>Science</i> (2019) [doi: 10.1126/science.aau8287]<p><b>7/18/18</b>: Nicolas D et al. <a href=https://doi.org/10.1073/pnas.1722330115 rel=noreferrer target=_blank>Modulation of transcriptional burst frequency by histone acetylation</a>. <i>PNAS</i> (2018) [doi: 10.1073/pnas.1722330115]<p><b>6/28/18</b>: Recillas-Targa F et al. <a href=https://doi.org/10.1073/pnas.102179399 rel=noreferrer target=_blank>Position-effect protection and enhancer blocking by the chicken beta-globin insulator are separable activities</a>. <i>PNAS</i> (2002) [doi: 10.1073/pnas.102179399]<p><b>6/20/18</b>: Diss G, Lehner B <a href=https://doi.org/10.7554/eLife.32472 rel=noreferrer target=_blank>The genetic landscape of a physical interaction</a>. <i>eLife</i> (2018) [doi: 10.7554/eLife.32472]<p><b>6/13/18</b>: Shen N et al. <a href=https://doi.org/10.1016/j.cels.2018.02.009 rel=noreferrer target=_blank>Divergence in DNA Specificity among Paralogous Transcription Factors Contributes to Their Differential In Vivo Binding</a>. <i>Cell Systems</i> (2018) [doi: 10.1016/j.cels.2018.02.009]<p><b>6/6/18</b>: Osterwalder M et al. <a href=https://doi.org/10.1038/nature25461 rel=noreferrer target=_blank>Enhancer redundancy provides phenotypic robustness in mammalian development</a>. <i>Nature</i> (2018) [doi: 10.1038/nature25461]<p><b>5/9/18</b>: Moretto F et al. <a href=https://doi.org/10.1038/s41467-018-03213-z rel=noreferrer target=_blank>A regulatory circuit of two lncRNAs and a master regulator directs cell fate in yeast</a>. <i>Nature Communications</i> (2018) [doi: 10.1038/s41467-018-03213-z]<p><b>5/2/18</b>: Batut PJ, Gingeras TR <a href=https://doi.org/10.7554/eLife.29005 rel=noreferrer target=_blank>Conserved noncoding transcription and core promoter regulatory code in early Drosophila development</a>. <i>eLife</i> (2017) [doi: 10.7554/eLife.29005]<p><b>4/25/18</b>: Ren G et al. <a href=https://doi.org/10.1016/j.molcel.2017.08.026 rel=noreferrer target=_blank>CTCF-Mediated Enhancer-Promoter Interaction Is a Critical Regulator of Cell-to-Cell Variation of Gene Expression</a>. <i>Molecular Cell</i> (2017) [doi: 10.1016/j.molcel.2017.08.026]<p><b>1/31/18</b>: Corrales M et al. <a href=https://doi.org/10.1101/gr.211433.116 rel=noreferrer target=_blank>Clustering of Drosophila housekeeping promoters facilitates their expression</a>. <i>Genome Research</i> (2017) [doi: 10.1101/gr.211433.116]<p><b>1/24/18</b>: Jiang SY et al. <a href=https://doi.org/10.1534/g3.117.300252 rel=noreferrer target=_blank>Dissecting Nucleosome Function with a Comprehensive Histone H2A and H2B Mutant Library</a>. <i>G3-Genes Genomes Genetics</i> (2017) [doi: 10.1534/g3.117.300252]<p><b>12/13/17</b>: Ye ZQ et al. <a href=https://doi.org/10.1093/nar/gkw659 rel=noreferrer target=_blank>Genome-wide analysis reveals positional-nucleosome-oriented binding pattern of pioneer factor FOXA1</a>. <i>Nucleic Acids Research</i> (2016) [doi: 10.1093/nar/gkw659]<p><b>2/10/16</b>: Burrows CK et al. <a href=https://doi.org/10.1371/journal.pgen.1005793 rel=noreferrer target=_blank>Genetic Variation, Not Cell Type of Origin, Underlies the Majority of Identifiable Regulatory Differences in iPSCs</a>. <i>PLOS Genetics</i> (2016) [doi: 10.1371/journal.pgen.1005793]<p><b>2/3/16</b>: Sorrells TR et al. <a href=https://doi.org/10.1038/nature14613 rel=noreferrer target=_blank>Intersecting transcription networks constrain gene regulatory evolution</a>. <i>Nature</i> (2015) [doi: 10.1038/nature14613]<p><b>1/27/16</b>: Kellogg RA et al. <a href=https://doi.org/10.7554/eLife.08931 rel=noreferrer target=_blank>Digital signaling decouples activation probability and population heterogeneity</a>. <i>eLife</i> (2015) [doi: 10.7554/eLife.08931]<p><b>1/20/16</b>: van Dijk D et al. <a href=https://doi.org/10.1101/033753 rel=noreferrer target=_blank>Competition between binding sites determines gene expression at low transcription factor concentrations</a>. <i>bioRχiv Preprint</i> (2016) [doi: 10.1101/033753]<p><b>12/16/15</b>: Jolma A et al. <a href=https://doi.org/10.1038/nature15518 rel=noreferrer target=_blank>DNA-dependent formation of transcription factor pairs alters their binding specificity</a>. <i>Nature</i> (2015) [doi: 10.1038/nature15518]<p><b>12/9/15</b>: Stampfel G et al. <a href=https://doi.org/10.1038/nature15545 rel=noreferrer target=_blank>Transcriptional regulators form diverse groups with context-dependent regulatory functions</a>. <i>Nature</i> (2015) [doi: 10.1038/nature15545]</div></main><footer><div><a href=https://bclab.wustl.edu>Cohen Lab</a> / <a href=http://genetics.wustl.edu/ rel=noreferrer>Department of Genetics</a> / <a href=https://genomesciences.wustl.edu/ rel=noreferrer>Center for Genome Sciences and Systems Biology</a> / <a href=https://medicine.wustl.edu/ rel=noreferrer>Washington University School of Medicine in St. Louis</a></div></footer></div><dialog id=lightbox><img alt id=lightbox-image></dialog><script>lightbox = document.getElementById("lightbox");
        lightboxImage = document.getElementById("lightbox-image");

        function showImageLightbox(anEvent) {
            anEvent.preventDefault();
            lightboxImage.setAttribute("src", this.getAttribute("href"));
            lightboxImage.setAttribute("alt", this.querySelector("img").getAttribute("alt"));
            lightbox.showModal();
        }

        lightbox.onclick = function(anEvent) {
            anEvent.preventDefault();
            lightboxImage.setAttribute("src", "");
            lightboxImage.setAttribute("alt", "");
            lightbox.close();
        };

        const lightboxImageLinks = document.querySelectorAll(".gallery a");

        for (var i = 0; i < lightboxImageLinks.length; i++) {
            lightboxImageLinks[i].onclick = showImageLightbox;
        }</script>